tests/langchain/sample_code/chain.py

```diff
@@ -84,3 +88,5 @@ def _llm_type(self) -> str:
     doc_uri="doc-uri",
     other_columns=["column1", "column2"],
 )
+
+retrieval_chain.invoke({"question": "What is the capital of Japan?"})
```

Comment:
```suggestion
retrieval_chain.invoke({"question": "What is the capital of Japan?"})
assert len(get_traces()) == 0
```

can we call `get_traces` to clarify why we're calling `invoke` here?

Reply:
Nope, this notebook mimics customers code, and it should generate traces if executed normally (we only suppress when it is loaded as model, so only assert in pytest)